R/tmb.R

```diff
@@ -148,6 +173,8 @@ h_mmrm_tmb_formula_parts <- function(formula) {
 #'     observed visits for each subjects. So the sum of this vector equals `n`.
 #' - `cov_type`: `string` value specifying the covariance type.
 #' - `reml`: `int` specifying whether REML estimation is used (1), otherwise ML (0).
+#' - `subject_groups`: `factor` or `integer` specifying the grouping for each subject.
```

Comment:
if possible, would be better to be strict here - so either take factor or integer, but not accept both. since this is an internal function. if for user we want to be nice, then can do that upstream and convert to factor or integer for here